content/recipes/reusability/miappe.md

```diff
@@ -0,0 +1,429 @@
+
+    Appropriate metadata for your read data submission should look similar to this:
+
+
+    * **Sample reference:** *<enter the BioSamples accession here>*
+    * **Instrument Model:** Illumina HiSeq 2500
+    * **Library Name:** barley_library_1
+    * **Library Source:** GENOMIC
+    * **Library Selection:** Restriction Digest
+    * **Library Strategy:** GBS
+    * **Library Layout:** SINGLE
+    * **First File Name:** HOR_1361_BRG.fastq.gz
+
+
+    **Hint**: If you did not note down your sample accession, you can safely visit the ‘**Samples**’ tab, copy the accession, then return to the New Submission tab: your progress will be saved.
+
+7. Click '**Submit**' and see if your submission validates successfully. If you encounter errors, try using the '**Download Template Spreadsheet'** button, open the file and check it in this plain text format; it can be easier to fix errors this way than in the interface. 
+
+
+    Note that the use of template spreadsheets is the best way to submit multiple datasets through this interface.
+
+8. If you manage to complete your submission, visit the 'Runs' tab to review your submission.
+
+### Step 3: Check if used reference genome assembly is available
+
+Is a GCF / GCA accession number available ? Check on [https://www.ebi.ac.uk/ena/browser](https://www.ebi.ac.uk/ena/browser).
+
+
+
+* If yes, proceed directly to VCF submission at step 4.
+* If no, [submit reference genome assembly](https://ena-docs.readthedocs.io/en/latest/submit/assembly/genome.html) to [INSDC](https://www.insdc.org/) (NCBI Genbank / EBML-EBI ENA / DDBJ) and wait until accession number is issued, then proceed to step 4.
+
+
+### Step 4: Analyse sequencing results and submit VCF file to EVA
+
+
+#### Step 4.1: Analyse sequencing results
+
+The bioinformatician performs the computational analysis, then the genotyping results are archived into the LIMS.
+
+
+#### Step 4.2: Prepare genotyping dataset for submission
+
+Refer to the publication [Recommendations for the formatting of VCF files to make plant genotyping data FAIR](https://doi.org/10.12688/f1000research.109080.1).
+
+In order to ensure interoperability of VCF files, the following VCF meta-information lines should be used:
+
+
+##### Obligatory meta-information line :
+
+**##fileformat** : file format. \
+\
+Example: \
+  `##fileformat=VCFv4.3  `
+
+
+##### Recommended meta-information lines :
+
+**##fileDate(Date)**: creation date of the VCF in the basic form without separator: YYYYMMDD \
+\
+Example: \
+  `##fileDate=20120921` 
+
+**##bioinformatics_source (URL or URI)**: Analytic approach usually consisting of chains of bioinformatics tools for creating the VCF file specified as the DOI of a publication, or more generally as URL/URI, like a public repository for the scripts used. \
```

Comment:
suggestion: state something about depositing Workflows to Workflowhub.eu , possibly in CWL format.

In so doing, we'll be able to link to the relevant Recipe about Workflows in the FAIR Cookbook . Tx